packages/design/src/Form/components/GenderId/index.tsx

```diff
@@ -0,0 +1,3 @@
+import { GenderIdPattern } from './GenderId.js';
+
+export default GenderIdPattern;
```

Comment:
This component has the import and immediate export convention, but the older patterns have the function body directly in the `index.tsx` file without the extra file. Could you make that change here?

Reply:
This totally makes sense and I agree on this! I will update it for gender id as part of this PR and will go ahead and file another PR for rest of the components.